.github/workflows/continuous-deployment.yml

```diff
@@ -48,13 +48,49 @@ jobs:
         with:
           args: "--skip-sorting-imports --replace"
 
+  security-sca-scan-frontend:
+    runs-on: ubuntu-latest
+    name: Scan repository frontend with Veracode SCA
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Run Veracode SCA for frontend
+        env:
+          SRCCLR_API_TOKEN: ${{ secrets.SRCCLR_API_TOKEN }}
+          SRCCLR_REGION: ${{ secrets.SRCCLR_REGION }}
+        uses: veracode/veracode-sca@v2.1.6
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          create-issues: false
+          path: gui/
+
+  security-sca-scan-backend:
+    runs-on: ubuntu-latest
+    name: Scan repository backend with Veracode SCA
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Run Veracode SCA for backend
+        env:
+          SRCCLR_API_TOKEN: ${{ secrets.SRCCLR_API_TOKEN }}
+          SRCCLR_REGION: ${{ secrets.SRCCLR_REGION }}
+        uses: veracode/veracode-sca@v2.1.6
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          create-issues: false
+          path: functions/node/
```

Comment:
path needs changed to backend/functions/

Reply:
paths are updated